docs/contributing.md

```diff
@@ -249,6 +253,45 @@ Now for the process itself:
 
    You're done!
 
+### Option B: Interactive Mode
+
+You can use the interactive web UI to streamline the release process by running:
+`yarn create-release-branch -i`
+
+This will:
+
+1. Start a local web server (default port 3000)
+
+2. Open a browser interface showing all packages with changes since their last release
+
+3. Allow you to visually select which packages to include in the release
+
+4. Provide instant validation of your selections, including:
+
+   - Identifying packages that need updates when their dependencies are being released
+   - Flagging packages that need to be included when their peer dependencies are being updated
+   - Validating version bumps against semantic versioning rules
+   - Ensuring all necessary dependent packages are included in the release
+
+5. Once your selections are complete, it will directly create the release branch with all necessary version bumps and changelog updates
+
+For example:
+
+- If you're releasing Package A that depends on Package B, the UI will prompt you to include Package B
+- If you're releasing Package B with breaking changes, the UI will identify any packages that have peer dependencies on Package B that need to be updated
+
+You can specify a different port if needed:
```

Comment:
We should automatically assign an open port so people don't have to worry about this, but this is fine to include for now.